a10_octavia/db/repositories.py

```diff
@@ -219,9 +218,8 @@ def getVThunderFromLB(self, session, lb_id):
 
     def getBackupVThunderFromLB(self, session, lb_id):
         model = session.query(self.model_class).filter(
-            self.model_class.loadbalancer_id == lb_id).filter(and_(self.model_class.status == "ACTIVE",
-                                                                   or_(self.model_class.role == "STANDALONE",
-                                                                       self.model_class.role == "BACKUP"))).first()
+            self.model_class.loadbalancer_id == lb_id).filter(or_(self.model_class.role == "STANDALONE",
```

Comment:
use `constants.ROLE_STANDALONE` instead

Reply:
use `constants.ROLE_BACKUP`

Reply:
To be taken care in code refactors of files